code/__HELPERS/unsorted.dm

```diff
@@ -1259,3 +1259,42 @@ will handle it, but:
 		pois[name] = M
 
 	return pois
+
+//Converts input direction to a list of either two same directions if the input is cardinal
+//Otherwise will return the two directions that make up the diagonal
+/proc/DiagonalToCardinal(direct)
```

Comment:
I would consider use of this sort of helper a code smell. There are better ways of iterating over dirs

Reply:
Proc no longer required